package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fragments-built-ins",
-  "version": "0.5.5",
+  "version": "0.5.7",
```

Comment:
The PR can just be the feature work. The bump can be committed directly to master on publish and doesn't need to be included here. Helps when multiple features may be in flight at once.